data/supporters.json

```diff
@@ -1429,6 +1429,27 @@
           "twitter": "_amatellanes"
         }
       ]
+    },
+    {
+      "name": "ThoughtWorks Tech Talks NYC Meetup",
```

Comment:
This was put into the `"conferences"` section of this file. It sounds like it might belong to `"usergroups"`. What do you think?

Reply:
my bad! thanks, I moved it now to `"usergroups"` :) and removed the hashtag link as suggested until #150 is done